Correctly recognize cssPseudo classes for indent

benbernard opened the pull request:
This fixes a problem where a section like this:

```
  .some-class
    &:hover
      color gray
```

if you invoked indention after the &:hover line, it would not indent a new level.

Hope this is the right way to contribute, let me know if there is a different policy on how to submit PRs, I have another change around flex attributes

wavded commented:
Thx @benbernard for the patch, the big thing is making sure `test.styl` looks good after your change.  Also, could you add this test case to `test.styl` so we have it documented?

benbernard commented:
Added a little bit to test.styl.

Problem is that I'm not sure how useful it is, since its not a thing that changes the look of the file, only changes the indent behavior on new lines.  Added a comment to indicate that.  Is there a better way to add the test case?

wavded commented:
@benbernard ahh gotcha sorry, I misread and thought it was syntax related too, either way, probably fine to have it in there.  Thx!
